### PR TITLE
Added API Endpoint for accessing state level maps

### PR DESCRIPTION
**Pull request by bengao01**

### CODE REVIEW

File: frontend/src/mapQuiz/mapQuiz.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ export class MapQuiz extends React.Component {
     }
 
     /**
+     * When this component is mounted to the DOM, get GeoJSON data from the server
```
Comment (ryaanahmed) [inline frontend/src/mapQuiz/mapQuiz.js]: this line is duplicated

File: frontend/src/index/index.js [hunk 2]
```diff
@@ -39,6 +39,14 @@ export class IndexView extends React.Component {
                     '
                 />
 
+                <IndexCard
+                    url='/campaign_game/'
+                    title='Campaign Game'
+                    description='
+                        Try to make a speech to get people to vote for you.
```
Comment (ryaanahmed) [inline frontend/src/index/index.js]: I think the "Try" might be in the wrong place in this sentence...

File: frontend/src/campaign/campaignView.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    render() {
+        if (!this.state.map_data) {
+            return (<div>Loading!</div>);
+
+        }
+        return (
+            <svg
+                height="800"
```
Comment (ryaanahmed) [inline frontend/src/campaign/campaignView.js]: use this.map_width and this.map_height here as well

File: frontend/src/campaign/campaignView.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                }
+            });
+            const geo_json = await map.json();
+            const map_data = project_features_and_create_svg_paths(geo_json, this.map_width,
+                this.map_height);
+            const input_tracker = this.initialize_input_tracker(map_data);
+            await this.setState({
+                map_data: map_data,
+                input_tracker: input_tracker,
+                // budgetData: JSON.parse(budgetData),
```
Comment (ryaanahmed) [inline frontend/src/campaign/campaignView.js:35]: remove cruft like this, plz!

File: frontend/src/campaign/campaignView.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        this.map_height = 800;
+        this.map_width = 800;
+    }
+
+    /**
+     * When this component is mounted to the DOM, get democracy score data from the server
+     */
+    async componentDidMount() {
+        try {
+            const map= await fetch('/api/state_map_geojson/ZAF/', {
```
Comment (ryaanahmed) [inline frontend/src/campaign/campaignView.js]: space around `=` (hmm... linter should have bitten you here; I'll take a look at why it didn't...)

**Review by ryaanahmed — CHANGES_REQUESTED** [review]
Requesting changes, mostly for nitpicks -- other than that looks good.

**Comment by ryaanahmed** [comment]
Wait, no, what am I talking about!? TESTS!

e.g., test that the list returned for `ZAF` is of length 9, etc. Would be good to get some quick tests in now so we don't break this with future refactors.

**Comment by bengao01** [comment]
I added the 3 test cases and fixed the previous comments, so I think everything should be good to merge now.

**Review by ryaanahmed — APPROVED** [review]
@bengao01 great tests!

Thanks for this example of how to test API calls using Django's built-in testing tools. That'll really help others write API endpoint tests moving forward.

merging.

**Pull request merged**